Suggested changes (#193)
federated-research/docs: website/pages/federated_research_patterns/egress.mdx

|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | **None**   | The egress being requested is approved by default, there can be no manual or automated egress check in place, or no process to mitigate disclosure risks.                                            |

Reviewer: ```suggestion
| **None**   | The egress being requested is approved by default, there are no manual or automated egress check in place, and no process to mitigate disclosure risks.                                            |
```

Local egress method: Manual  
Central egress method: Any  

This egress pattern is most likely with the [Traditional Federation Pattern](/federated_research_patterns/computational/traditional), mirroring the current process. All data leaving an individual TRE has a manual disclosure control check, regardless of whether any disclosure control is performed on the federated network.

## Semi-automated
Local egress method: Semi-automated  
Central egress method: Any  

> Emerging tools (such as [SACRO](https://github.com/AI-SDC)) allow for the semi-automated checking of results for a disclosure risk. The decision to release a result is still made by a human being and undertaken by the TRE, with the aim that this semi-automation can first warn researchers and output checkers ahead of time that a result may be disclosive and therefore accelerate any disclosure control assessments undertaken at the time of egress from the TRE. 

Reviewer: ```suggestion
Emerging tools (such as [SACRO](https://github.com/AI-SDC)) allow for the semi-automated checking of results for a disclosure risk. The decision to release a result is still made by a human and undertaken by the TRE, with the aim that this semi-automation can warn researchers and output checkers that a result may be disclosive before a result is released and therefore accelerate any disclosure control assessments undertaken at the time of egress from the TRE. 
```
